OloEditor/src/Panels/SceneHierarchyPanel.cpp

```
@@ -211,30 +389,191 @@ namespace OloEngine
+                    }
+                    if (m_CommandHistory)
+                    {
+                        m_CommandHistory->PushAlreadyExecuted(std::make_unique<ReparentEntityCommand>(
+                            m_Context, entity.GetUUID(), oldParentUUID, UUID(0)));
+                    }
+                }
+            }
+
+            ImGui::Separator();
+
+            if (ImGui::MenuItem("Save as Prefab"))
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)